newsfragments/271.deprecation.rst

```diff
@@ -0,0 +1 @@
+``web3.utils.abi.collapse_if_tuple()`` is now deprecated in favor of ``web3.utils.abi.get_normalized_abi_component_type()`` and will be removed in a later release.
```

Comment:
```suggestion
``eth_utils.abi.collapse_if_tuple()`` is now deprecated in favor of ``eth_utils.abi.get_normalized_abi_component_type()`` and will be removed in a later release.
```

Reply:
Once we decide if this PR is breaking, this could end up moving this to a new PR. Will correct the module reference.